agent/config/parse_windows.go

```diff
@@ -53,6 +53,28 @@ func parseGMSACapability() bool {
 	return false
 }
 
+// parseFSxWindowsFileServerCapability is used to determine if fsxWindowsFileServer support can be enabled
```

Comment:
This looks almost identical to `parseGMSACapability`? 
Can this be refactored to incorporate a common function that accepts an environment variable?  
Then both of these methods can utilize the common method and we can avoid duplication.

Reply:
will do